src/lib/layout/footer.svelte

```diff
@@ -1,47 +1,53 @@
-<footer class="main-footer">
-    <div class="container">
-        <ul class="inline-links is-center">
+<script>
```

Comment:
I think you got some footer from another PR in here 😉

Reply:
Oops! Should be fixed now